helmfile/charts/networkpolicy-workload/values.yaml

```
@@ -27,3 +27,18 @@ ingressNginx:
     enabled : false
     ips:
      - set-me-if-enabled
+
+coredns:
+  enabled: true
+  externalDns:
+    ips:
+      - "172.16.0.2/32"
+      - "172.16.0.3/32"
```

I guess this should be 
```suggestion
      - "0.0.0.0/0"
```

---

changed